src/tests/unit/fail_images/image_status_test_1.f90

```diff
@@ -0,0 +1,18 @@
+! Check the status of all images. Error only, when one unexpectedly failed.
+
+program test_image_status_1
+  use iso_fortran_env , only : STAT_STOPPED_IMAGE
```

Comment:
@vehre is there a reason you imported `STAT_STOPPED_IMAGE` but then never used it? Are you just testing if it's present?